Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py

```diff
@@ -530,6 +532,24 @@ def build_report_context(report_summary, upload_data, status, threshold):
             else:
                 context['DBotScore']['Score'] = 1
 
+        else:  # detonation did not return any data
+            # retrieve submission url by the task ID, if exist
+            submission_dt = demisto.dt(demisto.context(), 'ATD.Task(val.taskId === "{}")'.format(task_id))
+            if isinstance(submission_dt, list):
+                submission = submission_dt[0]
+            else:
+                submission = submission_dt
+            if isinstance(submission, dict):
+                if submission.get('url') and len(str(submission.get('url'))) > 0:
+                    context['DBotScore']['Type'] = 'application/url'
+                    context['DBotScore']['Indicator'] = submission.get('url')
+                else:  # if does not exist, submission is a file
+                    context['DBotScore']['Type'] = 'hash'
```

Comment:
I think you should assign the type hash only after you found the hash.(if it is sha1 or sha256)